OpenEdXMobile/src/main/java/org/edx/mobile/base/BaseFragment.java

```diff
@@ -44,21 +48,31 @@ public void onResume() {
     protected void onRevisit() {
     }
 
-    public void askForPermission(String permission, String[] permissions , int requestCode)
-    {
-        if (PermissionsUtil.checkPermissions(permission, getActivity())) {
-            if(permissionGranted != null) permissionGranted.onPermissionGranted(permissions , requestCode);
-        } else {
-            PermissionsUtil.requestPermissions(requestCode, permissions, BaseFragment.this);
+    /**
+     * When app needs run time permissions app calls this method to check and request permission if not granted
```

Comment:
Please read up how javadocs are written, this isn't following a proper javadoc style.